src/Datasource/Paging/PaginatedInterface.php

```diff
@@ -68,6 +68,15 @@ public function hasPrevPage(): bool;
      */
     public function hasNextPage(): bool;
 
+    /**
+     * Get the paginated items as an array.
+     *
+     * This will exhaust the iterator `items`.
+     *
+     * @return array
+     */
+    public function toArray(): array;
```

Comment:
Isn't that technically a breaking change since this method was added to the interface?

Reply:
Yeah, should it be an annotation at the top instead?

Reply:
:+1: Oops, I'll remove the interface update.